### DISCUSSION TITLE
Where is the basic documentation?

### DISCUSSION THREAD

**Discussion by paulehoffman** [marp-team/marp#505] [Q&A] [answered]
Greetings. Marp looks great, except that I can't find the basic documentation. I'm looking for a complete list of what settings can go in the first section, how the first page is different, and so on. I've searched all the repos, and the only thing I found was the short example on the main page.

**Comment by yhatt** [comment, marked as answer]
Duplicated: #503, #126

The documentation of the basic syntax is already available on https://marpit.marp.app/. Marpit is the base framework that Marp is depending on, and the things you are looking for could find at https://marpit.marp.app/directives.

And every marp-team repositories have detailed documentation on README.md.

**Reply by stdedos** [reply]
I have also "struggled" to find syntax help :sweat_smile:

Would you **kindly** add a header _there_

<img width="680" height="602" alt="image" src="https://github.com/user-attachments/assets/2edf9348-bd57-4796-a43c-0209cb0ae341" />

mentioning "Docs" or "Syntax" or _anything along those lines_? :sweat_smile: :pray: 

It is bit confusing UX-wise to have to visit the page, scroll all the way to the bottom, and click "Documentation" on the "Marpit framework" it sounds like an SDK rather than user-facing instructions / syntax help.
To me at least :sweat_smile:

> And every marp-team repositories have detailed documentation on README.md.

Also, yes, indeed, this https://github.com/marp-team/marp-cli#marp-teammarp-cli

> @marp-team/marp-cli
> CircleCI GitHub Workflow Status Codecov npm Docker LICENSE
>
> A CLI interface, for [Marp](https://github.com/marp-team/marp) (using [@marp-team/marp-core](https://github.com/marp-team/marp-core)) and any slide deck converter based on [Marpit](https://marpit.marp.app/) framework.

does contain a link to the "Marpit (Markdown) Syntax". But same as above: Idk why "Marpit framework" would "translate" as "Marpit (Markdown) Syntax" in everyone's head :pray:

**Reply by yhatt** [reply]
> it sounds like an SDK rather than user-facing instructions / syntax help.

Your understanding is correct. https://marpit.marp.app/ is treated as the SDK documentation.

Due to the redundancy of managing multiple documents and the high maintenance cost, writing a guide separately has lower priority than maintaining README, and the convention of referring to the SDK documentation by users continues.
(Especially since the maintainer is not native English speakers, I want dedicated time to write proper documentation for Marp but not now)

The current marp.app is a bit obsoleted, so we will manage the brand-new website that is including the current website content and a comprehensive guide in the future.

**Reply by stdedos** [reply]
I don't want anyone to maintain separate documents :sweat_smile: I am aware of the cost of maintaining documents (hence I write none :-p)

But still adding a button to the top saying "Syntax" and linking it to https://marpit.marp.app/markdown would be much more straightforward "where to look" to learn/use Marpit as an end user.

Thank you for taking the time to respond :pray:

**Reply by yhatt** [reply]
Thank you for your supplementation :)

Actually https://marpit.marp.app/markdown is not documentation for Marp, but for the Marpit framework. As the Marp team, we want to direct users to the user guide, not the SDK docs. That is precisely why I describe the importance of documentation.

Sneak peek: I'm working on the new Marp website that has a "Docs" menu for a comprehensive docs of Marp.

<img width="400" src="https://github.com/user-attachments/assets/2d839bdc-8b57-4d78-a6c5-aca31bb02052" />

**Reply by stdedos** [reply]
😍😍😍😍
That looks amazing 😀 

Okay, okay - I trust you ❤️
Godspeed, and tons of strength 💪 

(PS: If this, or part of this, would be a repo let me know - I might contribute small-size / low-friction changes when I see them 🙏)

**Reply by maagenta** [reply]
Looks amazing, much more cleaner  🔥

**Comment by bagustris** [comment]
Yes, marp lack of documentation. When I clicked "Getting Started," it pointed to Marp's tools and integration. I need to ask ChatGPT then, and it gives an answer: installing marp-cli and a simple example. It should be documented in a minimum step-by-step manner to make slides with Marp. Changing "Getting Started" to `https://marpit.marp.app/markdown` or adding a new  "Documentation" menu will be helpful.

**Reply by yhatt** [reply]
It has already planned to add docs into a brand-new website (See above: https://github.com/orgs/marp-team/discussions/505#discussioncomment-15013956). `https://marpit.marp.app/markdown` is not suited as a primary document because Marpit is highly SDK-centric, and lacking several features described in README of Marp Core.
